.gitignore

```diff
@@ -2,3 +2,4 @@ releases/
 !releases/.gitkeep
 vendor/
 /tmp/
+.idea
```

Comment:
Keep this in your global gitignore file.

Reply:
Ok